app/models/user.rb

```diff
@@ -6,7 +6,23 @@ class User < ActiveRecord::Base
   has_many :recommended_items
   has_many :television_shows, through: :recommended_items
 
+  has_many :following_relationships,
+    class_name: "Relationship", foreign_key: :follower_id
+  has_many :following, through: :following_relationships, source: :followed
```

Comment:
Every time I write this follower stuff, I feel like we expose some holes in the English language.